src/libsystemd/sd-bus/bus-creds.h

```diff
@@ -63,10 +67,22 @@ struct sd_bus_creds {
         char *description, *unescaped_description;
 };
 
+#define SD_BUS_CREDS_INIT_FIELDS        \
+        .uid = UID_INVALID,             \
+        .euid = UID_INVALID,            \
+        .suid = UID_INVALID,            \
+        .fsuid = UID_INVALID,           \
+        .gid = GID_INVALID,             \
+        .egid = GID_INVALID,            \
+        .sgid = GID_INVALID,            \
+        .fsgid = GID_INVALID,           \
+        .pidfd = -EBADF,                \
+        .audit_login_uid = UID_INVALID
```

Comment:
(I prefer to have a trailing comma here)

Reply:
that would mean we couldn't embed the macro easily and naturally when initializing things, as we'd like to insert the comma then, to separate it from what's coming next, and we'd have two commas there then...

Reply:
Ah, right. Sorry for the noise.

Reply:
(i.e. the idea is that you can use it like this:

```c
        *c = (sd_bus_creds) {
                .allocated = true,
                SD_BUS_CREDS_INIT_FIELDS,
                .n_ref = 1,
        };
```

and it would look really weird, if this would not have a comma after the macro use...